sidebars.js

```diff
@@ -85,6 +85,16 @@ const sidebars = {
           id: 'getting-started',
           label: 'Run a Contract Through ScalarDL Ledger',
         },
+        {
+          type: 'doc',
+          id: 'getting-started-hashstore',
+          label: 'Try Using ScalarDL HashStore',
+        },
+        {
+          type: 'doc',
+          id: 'getting-started-tablestore',
+          label: 'Try Using ScalarDL TableStore',
+        },
```

Comment:
@feeblefakie Since these are getting started docs, I added them to the `Quickstart` category. 

Do you think they're a bit too advanced for the `Quickstart` category and should be in the `Develop` category instead? In that case, we should probably rethink the title (like `Use ...` or something more appropriate for HashStore and TableStore).